core/src/kmx/kmx_plus.cpp

```diff
@@ -211,6 +211,18 @@ COMP_KMXPLUS_VKEY::valid(KMX_DWORD _kmn_unused(length)) const {
   return true;
 }
 
+bool
+COMP_KMXPLUS_DISP::valid(KMX_DWORD _kmn_unused(length)) const {
+  DebugLog("disp: count 0x%X\n", this->count);
+  if (header.size < sizeof(*this)+(sizeof(entries[0])*count)) {
```

Comment:
Using `this->count` on l216 and `count` on l217; let's be consistent :grin:

Reply:
Ts on the brain ?